Sources/Overload/OvEditor/src/OvEditor/Core/EditorActions.cpp

```diff
@@ -377,6 +377,21 @@ void OvEditor::Core::EditorActions::ResetLayout()
     DelayAction([this]() {m_context.uiManager->ResetLayout("Config\\layout.ini"); });
 }
 
+void OvEditor::Core::EditorActions::SaveLayout(const std::string& p_fileName)
+{
+	DelayAction([&]() {m_context.uiManager->SaveLayout(std::ref(p_fileName)); });
```

Comment:
`std::ref` is mostly to force passing a reference in cases where it's ambiguous (multiple function overloads). In this case it isn't necessary.

Reply:
Old shenanigans again was thinking a copy was done without it back then, forgot to remove, done!